src/fty_security_wallet.cc

```diff
@@ -34,15 +34,23 @@
 #include "fty_common_socket_basic_mailbox_server.h"
 
 #include <thread>
+#include <fty_service_status.h>
 
 //functions
 void usage();
 
 int main (int argc, char *argv [])
 {
+    //start service status
+    fty::ServiceStatusPluginWrapperCollection statusProviders(SECURITY_WALLET_AGENT);
+    int pluginLoaded = statusProviders.addAll("/usr/libexec/service_status_plugins/", std::regex(".*\\.so"));
```

Comment:
this path should be a config point, managed by autotools and configure call

Reply:
target the master branch?